src/arc.ts

```diff
@@ -566,6 +576,77 @@ export function getAccountIsEnabled(): boolean {
   return !!getWeb3Provider();
 }
 
+const ACCOUNTSTORAGEKEY = "currentAddress";
```

Comment:
nice to use underscores between words in constants

Reply:
Done